deploy/horizon-nginx-proxy/nginx.conf.tmpl

```diff
@@ -28,6 +28,7 @@ http {
       # http://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive
       proxy_http_version 1.1;
       proxy_set_header Connection "";
+      proxy_set_header Host   ${HOST};
```

Comment:
@Ronserruya what if you use the nginx variable `$host` instead of hardcoding the value?

Reply:
Or are you passing $host to ${HOST}, which in that case is a bit confusing

Reply:
You can either use $$host in the docker-compose file to pass $host to nginx, or hardcode the host, you can see my comment on the PR